linkerd/protocol/http/src/e2e/scala/io/buoyant/linkerd/protocol/HttpEndToEndTest.scala

```diff
@@ -842,6 +842,107 @@ class HttpEndToEndTest
     }
   }
 
+  test("serverSession idleTimeMs should close server connections") {
+    val config =
+      s"""|routers:
+          |- protocol: http
+          |  experimental: true
+          |  dtab: |
+          |    /p/fox => /$$/inet/127.1/{fox.port} ;
+          |    /svc/century => /p/fox ;
+          |  servers:
+          |  - port: 0
+          |    serverSession:
+          |      idleTimeMs: 1500
+          |      lifeTimeMs: 3000
+          |""".stripMargin
+
+    serverIdleTimeMsBaseTest(config){ (router:Router.Initialized, stats:InMemoryStatsReceiver) =>
+      // Assert
+      def connectsCount = () => stats.counters(Seq("rt", "http", "server", "127.0.0.1/0", "connects"))
```

Comment:
I don't think this needs to be a closure, I think it can simply be a def, similarly to how metrics are checked elsewhere in this file:

```
def connectsCount = stats.counters(Seq("rt", "http", "server", "127.0.0.1/0", "connects"))
```

Reply:
Thanks @adleong!
I updated the PR.
CircleCI seems to have failed on downloading a zookeeper dependency, could you please trigger a rebuild?